config/webpack.config.js

```diff
@@ -11,12 +11,10 @@ const dir_build = path.resolve(__dirname, basePath, 'build');
 
 module.exports = {
     entry: {
-        main: path.resolve(dir_js, 'main.js'),
-        vendor: [
-            'babel-polyfill',
```

Comment:
It seems kinda polyfill/ponyfill is going to be definitely needed when it comes to use some es6 features in browser runtime:
https://medium.com/@jcse/clearing-up-the-babel-6-ecosystem-c7678a314bf3
https://ponyfoo.com/articles/polyfills-or-ponyfills

Then I'm gonna create another PR changing some of the app code (e.g. by using Promises) in order to check which one of 'babel-polyfill'/'babel-runtime' should be more convenient in this project.